BeardedSpice/MediaStrategies/Youtube.js

```diff
@@ -4,16 +4,17 @@
 //
 //  Created by Jose Falcon on 12/15/13.
 //  Updated by Alin Panaitiu on 3/2/18.
+//  Updated by Vladislav Gapurov on 07/28/18
 //  Copyright (c) 2013 Tyler Rhodes / Jose Falcon. All rights reserved.
 //
 
 BSStrategy = {
-  version: 3,
+  version: 4,
   displayName: "Youtube",
   accepts: {
     method: "predicateOnTab",
-    format: "%K LIKE[c] '*youtube.com/watch*'",
-    args: ["URL"]
+    format: "%K LIKE[c] '*youtube.com/watch*' && !%@ LIKE[c] '*music.youtube.com*'",
```

Comment:
(I know nothing of Objective-C.) What is the difference between `%K` and `%@` in `NSPredicate`?

Figured I might as well Google it:
> - `%@` is a var arg substitution for an object value—often a string, number, or date.
> - `%K` is a var arg substitution for a key path.
>
> ([source](https://nshipster.com/nspredicate/))

(Doesn't makes things much clearer for me, to be honest. 😅 )

Seeing how `%K` is used in all strategies, would it make sense to keep it the same?